jdaviz/configs/cubeviz/plugins/slice/slice.py

```diff
@@ -134,12 +141,11 @@ def _on_setting_changed(self, event):
 
     @observe('slider')
     def _on_slider_updated(self, event):
-        value = int(event.get('new', 0))
+        value = int(event.get('new', int(len(self._x_all)/2)))
```

Comment:
probably will never use the default here, but might as well default to the middle since that is the initial default position.